testing/features/support/components/search/standard.rb

```diff
@@ -18,7 +18,7 @@ def search_for(term)
     private
 
     def clear_field
-      search_field.send_keys(:end)
+      search_field.send_keys(%i[end])
```

Comment:
Either we both have or I have seen this error before.

I think going forward we just need to be mindful to always use arrays for this method. I'll raise a minor issue in ruby